packages/react-dialog/src/stories/AlertDialog.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { Button } from '@fluentui/react-button';
+import { Dialog, DialogProps, DialogHeader, DialogBody, DialogFooter } from '../index';
+
+export const AlertDialog = (props: Partial<DialogProps>) => {
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  return (
+    <>
+      <Button
+        onClick={() => {
+          setIsOpen(true);
+        }}
+        aria-haspopup="true"
+      >
+        Open Dialog
+      </Button>
+
+      <Dialog type="alert" aria-labelledby="dialog-title" open={isOpen} {...props}>
```

Comment:
is there a way how we can enforce usage of label? For example changing the prop interface to require either aria-label or aria-labelledby?